### DISCUSSION TITLE
enable_diagnostics="false" => diag.enabled="false" not in schema.xml

### DISCUSSION THREAD

**Discussion by qeepcologne** [belaban/JGroups#623] [General]
changed enable_diagnostics="false" with diag.enabled="false" as in #597.
But now xml validation fails:

``
XML validation started.
Checking file:/home/bwahlen/repos/backend/remoting/src/test/resources/META-INF/jgroups5-cluster-test.xml...
Referenced entity at "http://www.jgroups.org/schema/jgroups-5.2.xsd".
Referenced entity at "http://www.jgroups.org/schema/relay.xsd".
Referenced entity at "http://www.jgroups.org/schema/fork-stacks.xsd".
cvc-complex-type.3.2.2: Attribute 'diag.enabled' is not allowed to appear in element 'UDP'. [17] 
XML validation finished.
``

i am using the latest schema http://www.jgroups.org/schema/jgroups-5.2.xsd
please update schema also.

**Comment by belaban** [comment]
Will do so. Meanwhile, create a local schema with `ant make-schema`

**Comment by belaban** [comment]
I uploaded the schema and the URL posted above should work now.
